### ISSUE TITLE
[2.6.0] [Enhancement] Button for "Katheterwechsel" could be changed

### ISSUE THREAD

**Issue by fezulin** [MilosKozak/AndroidAPS#2491]
This is no description of a bug, but a suggestion for change, that might be an enhancement.
In AAPS 2.5.x there was a buttons for changing the insulin cartridge and another button for changing the cannula.
In AAPS 2.6.0 both functions can be reached with the same button. Currently the button shows the icon that was formerly used for cannula change.
I like to suggest, that both icons will be merged on the button. 
For German GUI the word “KATHETERWECHEL” does no longer perfectly fit. Other options might be “RESERVOIR / KATHETER” or simply “WECHSEL”.
Find attached a suggestion for the changed button.
If you choose not to change the button or to change it in another way – I am totally OK with that. 
:o)
![newButton](https://user-images.githubusercontent.com/16593716/76173296-7f7f5600-619e-11ea-8111-c38719175dac.png)
![newButtonInGUI](https://user-images.githubusercontent.com/16593716/76173297-85753700-619e-11ea-9b36-60bddc5770f8.png)

**Comment by fezulin** [comment]
Well... if you like to go a step further:
Another idea: How about merging the button for "Pump Battery Change", too?

There might be three checkboxes:
* change cannulla
* change cartridge
* change Pump Battery

The Button to reach these functions might look like this:
![newButtonB](https://user-images.githubusercontent.com/16593716/76219982-fe17da00-6216-11ea-95a3-7f2956e245e5.png)
![newButtonB_inGUI](https://user-images.githubusercontent.com/16593716/76219994-01ab6100-6217-11ea-9443-c9b938d03be4.png)
![NewButtonB2](https://user-images.githubusercontent.com/16593716/76220018-0a039c00-6217-11ea-88f8-de04d39eab2d.png)

**Comment by guydavies** [comment]
I don't like that idea. The catheter and reservoir changes are commonly
done together. The pump battery change is completely independent. Please
don't do that.

**Comment by fezulin** [comment]
@guydavies: You are right. “Pump battery change” is really something else. It would not be good, to bring these functions together. Thank you for pointing out this.

Depending on type of catheter: "Catheter change" and "reservoir change" are independent of eachother, too. This applies for “Accu check Rapid D link” and I guess there are other catheters with similar conditions.
If I had a wish for free: I would prefer to have separate buttons for these functions. If the these functions shall be merged in one button, I would prefer an icon that shows both: reservoir and catheter.
But I am aware: This is not a big issue at all. It is just a matter of “I would prefer if…”

**Comment by guydavies** [comment]
Thanks for pointing that out. I've only ever used systems where the two
were normally done simultaneously. Having them separate might be more
intuitive for some users.
